java-squid/src/main/java/org/sonar/java/model/VisitorsBridge.java

```diff
@@ -281,10 +281,10 @@ public void addIssue(File file, JavaCheck check, int line, String message) {
 
     /**
      *
-     * DO NOT GO ON RELEASE WITH THIS CONSTANT SET TO TRUE
+     * FIXME(mpaladin) DO NOT GO ON RELEASE WITH THIS CONSTANT SET TO TRUE
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1134)